### DISCUSSION TITLE
Small code mod I

### DISCUSSION THREAD

**Discussion by Mabuse1337** [datavorous/yars#3] [General]
Hey, great little program. I ended up wanting to scape all of a particular user's posts, so I added some further items to the category field on the subreddit scraper to run on /u/ instead of /r/ and then I can just put in a username instead of a subreddit.

So for instance, for category 'usernew', 
url = f"https://www.reddit.com/user/{subreddit}/submitted/new.json"

and etc for the other categories.

I'm not great at using Github, but I thought I'd mention it since it's a quick add and a useful function.

**Comment by datavorous** [comment]
@Nabby109 that's great! Would you mind making a pull request?

**Reply by Mabuse1337** [reply]
Took me a bit to figure out how - still find the github interface confusing. Edited the code from your current yars.py post switch to uv and created a pull request for it. 

Recommend pointing out that for people like me who want to download attached images from all or a lot of a user's posts, to change the number of images to download to something larger. (Possibly just make it another variable that stands out to be edited more easily.)

# Attempt to download images from the first few posts
"for idx, post in enumerate(subreddit_posts[:num_images]" for instance.

**Reply by datavorous** [reply]
Alright, I'll look into it! Thanks for your contribution.
